skll/utilities/generate_predictions.py

```diff
@@ -103,41 +126,53 @@ def main(argv=None):
         formatter_class=argparse.ArgumentDefaultsHelpFormatter,
         conflict_handler='resolve')
     parser.add_argument('model_file',
-                        help='Model file to load and use for generating \
-                              predictions.')
-    parser.add_argument('input_file',
-                        help='A csv file, json file, or megam file \
-                              (with or without the label column), \
-                              with the appropriate suffix.',
+                        help='Model file to load and use for generating '
+                             'predictions.')
+    parser.add_argument('input_files',
```

Comment:
Just wondering, what's this change all about? It seems to be unrelated to the issue.